src/main/java/brandkon/brand/BrandRestController.java

```diff
@@ -0,0 +1,31 @@
+package brandkon.brand;
+
+import brandkon.brand.DTO.BrandResponseDto;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.bind.annotation.RestController;
+
+import java.util.List;
+
+@RestController
+public class BrandRestController {
+
+    private BrandService brandService;
+
+    public BrandRestController(BrandService brandService) {
+        this.brandService = brandService;
+    }
+
+    // 브랜드 목록 조회
+    @GetMapping("/brands")
+    public List<BrandResponseDto> findBrands(@RequestParam("category") String category) {
+        return brandService.findBrands(category);
+    }
+
+    // 브랜드 상세 조회
+    @GetMapping("/brands/{brandId}")
+    public BrandResponseDto brand(@PathVariable long brandId) {
```

Comment:
```suggestion
    public BrandResponse findById(@PathVariable long brandId) {
```
함수 이름을 좀 더 명확하게 짓는 것이 좋을 것 같아요